helmfile/values/opensearch/secrets.yaml.gotmpl

```diff
@@ -0,0 +1,51 @@
+opensearch:
```

Comment:
Shouldn't we have the objectstorage type, check here instead? Or does it work without one?

Reply:
It works without one if it runs without snapshots.